go/libkb/sig_chain.go

```diff
@@ -545,8 +544,17 @@ func isSubchainStart(currentLink *ChainLink, prevLink *ChainLink) bool {
 	if !currentLink.ToEldestKID().Equal(prevLink.ToEldestKID()) {
 		return true
 	}
+
 	// case 4
-	return hardcodedResets[currentLink.unpacked.sigID]
+	found, _, err := currentLink.checkSpecialLinksTable(hardcodedResets, uid, "harcoded resets")
+	if err != nil {
+		m.CWarningf("Error in isSubchainStart, will be ignored: %s", err.Error())
+		return false
```

Comment:
why ignore rather than bail out?

Reply:
i was going down that path but then i'd have to change a lot of callers that are in boolean conjunctions.  I can do it though, but it does mitigate the attack as is.

Reply:
(a lot = ~3)